README.md

```diff
@@ -10,11 +10,27 @@ A Python client for working with [STAC](https://stacspec.org/) APIs.
 
 ## Installation
 
-Install from PyPi.
-Other than [PySTAC](https://pystac.readthedocs.io) itself, the only dependencies for **pystac-client** are the Python [requests](https://docs.python-requests.org) and [dateutil](https://dateutil.readthedocs.io) libraries.
+PySTAC Client is published to PyPi as [pystac-client](https://pypi.org/project/pystac-client/).
+
+The only direct Python dependencies of **pystac-client** are [PySTAC](https://pystac.readthedocs.io),
+[requests](https://docs.python-requests.org), and [dateutil](https://dateutil.readthedocs.io).
+
+pip:
+
+```shell
+pip install pystac-client
+```
+
+uv:
+
+```shell
+uv add pystac-client
+```
+
+poetry:
 
 ```shell
-python -m pip install pystac-client
+poetry add pystac-client
```

Comment:
There's a balance to strike here between helping folks get set up, and having to enumerate all the ways someone might depend on this package (e.g. should we include **conda** instructions? **pipenv**?). My instinct is to _only_ provide `pip` instructions and assume that if you're using **poetry** or **uv** you know what you're doing and don't need our help?

Reply:
Yeah, I thought about conda too, and then was like, should we even have it at all? I feel like we should actually just document what the pypi package (plus optionals) are named, and assume that the a user will know how to install it in their local environment or add as a dependency of their project.

Reply:
Yeah I think we should only specify the pip instructions.